docs/modules/ROOT/pages/how-tos/configuring/overriding-compute-resources.adoc

```diff
@@ -0,0 +1,87 @@
+= Overriding compute resources
+
+_For background on compute resources, see link:https://tekton.dev/docs/pipelines/compute-resources/[Compute Resources in Tekton]._
+
+{ProductName} tasks have default resource requests and limits.
+Some set them explicitly in the Task definition,
+others inherit them from the link:https://tekton.dev/docs/pipelines/compute-resources/#limitrange-support[LimitRange] in your namespace.
+
+If the default resources do not meet your needs, you can take advantage of Tekton's
+link:https://tekton.dev/docs/pipelines/pipelineruns/#specifying-taskrunspecs[taskRunSpecs] to override them.
+You can configure `taskRunSpecs` in the PipelineRun files in your `.tekton` directory. For example:
+
+[source,yaml]
+----
+kind: PipelineRun
+spec:
+  taskRunSpecs:
+    - pipelineTaskName: build-container
+      stepSpecs:
+        - name: build
+          computeResources:
+            requests:
+              memory: 10Gi
+            limits:
+              memory: 10Gi
+
+    - pipelineTaskName: prefetch-dependencies
+      computeResources:
+        requests:
+          cpu: '1'
+          memory: 2Gi
+        limits:
+          memory: 2Gi
+
+  pipelineSpec:
+    tasks:
+      # ...
+      - name: prefetch-dependencies
+        taskRef: ...
+      # ...
+      - name: build-container
+        taskRef:
+          resolver: bundles
+          params:
+            - name: kind
+              value: task
+            - name: name
+              value: buildah
+            - name: bundle
+              value: quay.io/konflux-ci/tekton-catalog/task-buildah:0.2@sha256:3f0913dfb85e9aeb9916e412d10329528ddf4c8fba9958cba5291ca8ee247f7e
+----
+
+This PipelineRun:
+
+* Overrides the resources for the `build` step in the `build-container` task.
+  Other steps in the `build-container` task are not affected.
+* Sets the overall resources for the `prefetch-dependencies` task.
+  Tekton divides the resources equally among all the steps in the task.
+  If the prefetch task has two steps, each will receive `500m` cpu and `1Gi` memory.
+
+[TIP]
+====
+When overriding the resources for a specific step, you need to know the name of the step.
+You can find the names of all steps in the Task definition.
+If your Pipeline references the task as a bundle (like the example above), you can use `tkn bundle list` to see the Task definition:
+
+[source]
+----
+tkn bundle list -o yaml quay.io/konflux-ci/tekton-catalog/task-buildah:0.2@sha256:3f0913dfb85e9aeb9916e412d10329528ddf4c8fba9958cba5291ca8ee247f7e
+----
+
+====
+
+[NOTE]
+====
+To set the overall resources, we use
+link:https://tekton.dev/docs/pipelines/compute-resources/#task-level-compute-resources-configuration[task-level computeResources].
+To set the resources for a specific step, we use
+link:https://tekton.dev/docs/pipelines/taskruns/#configuring-task-steps-and-sidecars-in-a-taskrun[stepSpecs].
+
+As of Tekton Pipelines `v0.61.0`, these are a Beta feature and an Alpha feature, respectively.
```

Comment:
The respectively refers to computeResources and stepSpecs? Can we make that relationship clearer?